other_recipes/jellof rice.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+   <h1>Jellof Rice</h1> 
+   <img src="hanxiao-lQN1CVzgQUw-unsplash.jpg"width="50%">
```

Comment:
- [ ] Every image should have an `alt` attribute. Read about `<img>` [tags here](https://www.w3schools.com/html/html_images.asp).